### PR TITLE
Show units used when streaming the map over network. Fix #3328

### LINKED ISSUE #3328: Adding kb/bytes to 'Downloading map ...'
When you join a game in multiplayer, you of course need to download the map which is indicated by STR_5759, but it doesn't tell the player 'how much' it downloads
It could be bytes, kb, mb, TB, or meatballs

Just simply adding kb/byte (Obviously the correct value) would help people understand what the numbers mean

`STR_5759    :Downloading map ... ({INT32} / {INT32})` **`KB`**

### PR DESCRIPTION
**Pull request by janisozaur**
I changed the chunk size, so it could use some testing.

### CODE REVIEW

**Comment by marijnvdwerf** [comment]
Kibibytes (KiB), not kilobytes then ;)

File: data/language/english_uk.txt [hunk 1]
```diff
@@ -4064,7 +4064,7 @@ STR_5755    :Incorrect Software Version
 STR_5756    :Bad Password
 STR_5757    :Server Full
 STR_5758    :{OUTLINE}{GREEN}{STRING} has joined the game
-STR_5759    :Downloading map ... ({INT32} / {INT32})
+STR_5759    :Downloading map ... ({INT32} / {INT32}) kB
```
Comment (IntelOrca) [inline data/language/english_uk.txt]: KiB

**Comment by janisozaur** [comment]
updated

**Comment by IntelOrca** [comment]
You need to change 1000 to 1024 in ```Client_Handle_MAP```.

**Comment by janisozaur** [comment]
done

**Pull request merged**